src/encoded/static/components/matrix.js

```diff
@@ -7,6 +7,7 @@ import { svgIcon } from '../libs/svg-icons';
 import * as globals from './globals';
 import { BrowserSelector } from './objectutils';
 import { BatchDownload, FacetList, TextFilter } from './search';
+import pluralize from 'pluralize';
```

Comment:
This still needs to be put above underscore.

Reply:
If I am not mistaken, the above is old and was fixed yesterday. However, I did not update auditmatrix.js. That has been fixed.